unified-codes/apps/data-service/src/app/api.ts

```diff
@@ -11,12 +11,13 @@ import { KeyCloakIdentityProvider } from './IdentityProvider';
 export const createApolloServer = (_typeDefs, _resolvers, _dataSources, _authenticator) => {
   const AUTH_URL = 'http://127.0.0.1:9990/auth/realms/unified-codes';
 
+  const getDataSources = () => ({
+    DgraphDataSource: new Data.DgraphDataSource(),
+    RxNavDataSource: new Data.RxNavDataSource(),
+  });
```

Comment:
Hmm, idea is for these to be defaults (but they should be lifted out, actually!). 

In either case, could not get away with:

```
const dataSources = _dataSources ?? { 
  DgraphDataSource: new Data.DgraphDataSource(),
  RxNavDataSource: new RxNavDataSource()
};
```

Just to avoid the inline function?

Reply:
Also, I wanted to make the imports look pretty, but should probably have imported these as named imports!

```
import * as Schema from './schema';
import * as Resolvers from './resolvers';

import { DgraphDataSource, RxNavDataSource } from './data';
```

Reply:
> In either case, could not get away with:

no - that's what I tried first. Won't allow it.

Reply:
bah! Tried it now and it does work?!? Did the same thing yesterday and it wouldn't compile 🤷‍♂️